src/lib/pmd/PmdCommandInfo.ts

```diff
@@ -0,0 +1,42 @@
+import {PMD6_LIB, PMD6_VERSION} from "../../Constants";
+import path = require('path');
```

Comment:
We've been trying to keep up with newer Typescript syntax so that upgrading versions is easier.
https://stackoverflow.com/questions/29596714/new-es6-syntax-for-importing-commonjs-amd-modules-i-e-import-foo-require

Reply:
Sure.